src/workerd/api/crypto-impl-asymmetric.c++

```diff
@@ -2126,9 +2117,6 @@ private:
     jwk.kty = kj::str("OKP");
     jwk.crv = kj::str(getAlgorithmName() == "X25519"_kj ? "X25519"_kj : "Ed25519"_kj);
     jwk.x = kj::encodeBase64Url(kj::arrayPtr(rawPublicKey, publicKeyLen));
-    if (getAlgorithmName() == "Ed25519"_kj) {
-      jwk.alg = kj::str("EdDSA");
-    }
```

Comment:
There's a chance this could end up being a breaking change. Will need to determine if a compatibility flag is necessary on this particular bit.

Reply:
Cool, by all means please take this effort over, my hope here was merely to bring this to your attention ;)